usecase/authz_proxyd.go

```diff
@@ -0,0 +1,152 @@
+/*
```

Comment:
cannot trace what is changed in this file...

Reply:
not sure why github cannot display after the rename...

ignore last rename commit:
https://github.com/yahoojapan/authorization-proxy/pull/25/files/92d485daaf91fc7a2b2b688962bbd1d586ef5262#diff-098eecd860ad41b7422405e377d9c2a8

Reply:
why don't you use git mv command?

Reply:
I can see git mv in source tree, just cannot work in github

Reply:
maybe one thing, can you try to fix it?

Reply:
fix what?